tigerpath/majors_and_certificates/majors/ARC_2018.json

```
@@ -0,0 +1,201 @@
+{
+  "type": "Major",
+  "name": "Architecture",
+  "code": "ARC",
+  "degree": "AB",
+  "year": 2018,
+  "urls": [
+    "https://ua.princeton.edu/academic-units/school-architecture",
+    "http://soa.princeton.edu/"
+  ],
+  "contacts": [
+    {
+      "type": "Dean",
+      "name": "Mónica Ponce de León",
+      "email": "mpdl@princeton.edu"
+    },
+    {
+      "type": "Associate Dean",
+      "name": "Paul Lewis",
+      "email": "paul@ltlarchitects.com"
+    },
+    {
+      "type": "Director of Undergraduate Studies",
+      "name": "Mario I. Gandelsonas",
+      "email": "mgndlsns@princeton.edu"
+    }

```

From the department website, it looks to me like they alternate between the two of them each semester, Fall/Spring. Might be good to verify this, though.
```suggestion
    {
      "type": "Director of Undergraduate Studies",
      "name": "Mario I. Gandelsonas",
      "email": "mgndlsns@princeton.edu"
    },
    {
      "type": "Director of Undergraduate Studies",
      "name": "Cameron Wu",
      "email": "cwu@princeton.edu"
    }
```